src/management/apps.spec.ts

```diff
@@ -1,7 +1,9 @@
+import * as mockfs from "mock-fs";
 import { expect } from "chai";
 import * as sinon from "sinon";
 import * as fs from "fs";
 import * as nock from "nock";
+import FileSystem from "mock-fs/lib/filesystem";
```

Comment:
Missed this on the last pass, but its generally bad practice to dip into the lib of our imports. If this is not an exported type that you can get from mockfs, we should probably just use Record<string, any> instead in its place